crates/stark-felt/src/lib.rs

```diff
@@ -382,22 +486,55 @@ mod formatting {
 
     /// Represents [Felt] in decimal by default.
     impl fmt::Display for Felt {
```

Comment:
Should we display decimal or hexadecimal here?

Reply:
We have `LowerHex` & `UpperHex` implementations so it makes sense to display decimal here imo